src/Stratis.Bitcoin.IntegrationTests/BlockStore/ReOrgRegularlySpecification.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Stratis.Bitcoin.IntegrationTests.TestFramework;
+using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.BlockStore
+{
+    public partial class ReOrgRegularlySpecification : BddSpecification
+    {
+        [Fact]
+        public void A_selfishly_mining_node_broadcasts_longer_chain_causing_reorg()
+        {
+            Given(four_nodes);
+            And(each_mine_a_blocks);
+            And(mining_continues_to_maturity_to_allow_spend);
+            And(selfish_miner_disconnects_and_mines_10_blocks);
+            And(second_node_creates_a_transaction_and_broadcasts);
+            And(third_node_mines_this_block);
+            And(fouth_node_confirms_it_ensures_tx_present);
+            When(selfish_node_reconnects_and_broadcasts);
+            Then(second_third_and_fourth_node_reorg_to_longest_chain);
+            And(transaction_from_shorter_chain_is_missing);
+            // And(transaction_is_returned_to_the_mem_pool); // TODO: not implemented in the production code yet
```

Comment:
Is this still the case ?

Reply:
Yeah it is a tricky one - I would almost like this to be a failing test to make us implement it, but it depends on priorities.

Perhaps I keep a branch with it in ready for when implementing but remove here as it isn't the current behaviour of the system. Sound good?

Reply:
Keep it like this its good